src/app/public/modules/icon/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './icon-stack-item';
+export * from './icon.component';
+export * from './icon.module';
```

Comment:
Were these exports intentional? My understanding is we only bundle exports at the `public_api.ts` level to avoid dependency issues.

Reply:
Work began on this feature in SKY UX 3, so these additions are holdovers from that time. I'll remove them.